src/pages/questionPage.js

```diff
@@ -24,6 +24,18 @@ export const initQuestionPage = () => {
   for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
     const answerElement = createAnswerElement(key, answerText);
     answersListElement.appendChild(answerElement);
+
+    // making answers clickable
```

Comment:
Liked this comment! It's a good practice to inform the reader with those kind of comments.